### PR TITLE
Bump version 0.9.0

### PR DESCRIPTION
**Pull request by kmuehlbauer**
- remove scipy pinning

### CODE REVIEW

**Comment by kmuehlbauer** [comment]
@ocefpaf I've no idea why the tarball isn't found. I uploaded the version (https://pypi.python.org/pypi/wradlib/0.9.0) and can download it from the webfrontend. But the link doesn't seem to work.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,25 +7,25 @@ package:
 source:
     fn: wradlib-{{ version }}.tar.gz
     url: https://pypi.python.org/packages/source/w/wradlib/wradlib-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @kmuehlbauer change this to:

`https://pypi.io/packages/source/w/wradlib/wradlib-{{ version }}.tar.gz`

The old PyPI URLs no longer work. You can be re-direct to the new ones sometimes, but those are long and ugly. We are using the `pypi.io` for now.

**Comment by kmuehlbauer** [comment]
Thanks @ocefpaf.

**Comment by kmuehlbauer** [comment]
This worked now fine and compiled as expected. 

@ocefpaf @jjhelmus Can someone merge this,  not sure if self merges are appropriate in such cases.

**Comment by ocefpaf** [comment]
> @ocefpaf @jjhelmus Can someone merge this, not sure if self merges are appropriate in such cases.

It is. Thanks for the new version @kmuehlbauer!

**Pull request merged**